inst/tests/froll.Rraw

```diff
@@ -1450,7 +1450,7 @@ test(6001.729, frollvar(adaptive=TRUE, c(1:2,NA), c(2,0,2), algo="exact", na.rm=
 test(6001.730, frollvar(adaptive=TRUE, c(1:2,NA), c(2,0,2), fill=99, algo="exact", na.rm=TRUE), c(99,NA,NA))
 y = c(1e8+2.980232e-8, 1e8, 1e8, 1e8) # CLAMP0 test
 test(6001.731, frollvar(y, 3)[4L], 0)
-test(6001.732, frollsd(y, 3)[4L], 0)
+if (exact_NaN) test(6001.732, frollsd(y, 3)[4L], 0)
```

Comment:
the other cases are all using `algo='exact'`, so this would be the first case being skipped with `algo='fast'`

Reply:
it's also v. interesting that `frollvar(y, 3)[4L]`, which per glancing the implementation only differs by applying `sqrt()`, does not have this numerical difference issue

Reply:
Yes, sqrt must be causing this rounding error

Reply:
Any idea why `2.980232e-8` was chosen? Maybe we could make that a function of `.Machine$double.eps` (assuming that changes on/off valgrind) or of `.Machine$longdouble.digits`?

Reply:
No idea...